src/components/UNSTABLE__PageLayouts/Detail.stories.js

```diff
@@ -47,187 +48,197 @@ export default {
+      <DescriptionModule>
+        {({ getLayoutProps }) => (
+          <p {...getLayoutProps()}>
+            BadFlick is a backdoor that is usually seen being distributed using
+            exploited word documents. It does not have any persistence to
+            survive reboot, but it is capable of opening a reverse shell
+            connection to its C2 server where it can download and execute
+            possibly other malware.
+          </p>
+        )}
+      </DescriptionModule>
+    </div>
+
+    <div
+      className={`${settings.prefix}--col-start-9 ${settings.prefix}--col-span-8`}
+    >
+      <DescriptionListModule>
+        <TypeLayout>
+          <TypeLayoutBody>
+            <TypeLayoutRow>
+              <TypeLayoutCell>Created by</TypeLayoutCell>
+              <TypeLayoutCell>X-Force IRIS</TypeLayoutCell>
+            </TypeLayoutRow>
+            <TypeLayoutRow>
+              <TypeLayoutCell>Last updated</TypeLayoutCell>
+              <TypeLayoutCell>Jul 14 2019</TypeLayoutCell>
+            </TypeLayoutRow>
+          </TypeLayoutBody>
+        </TypeLayout>
+      </DescriptionListModule>
+    </div>
+
+    <div className={`${settings.prefix}--col-span-100`}>
+      <TitleBarModule title="Related reports" />
+    </div>
+
+    <div className={`${settings.prefix}--col-span-100`}>
+      <ActionBarModule>
+        Supplementary details
+        <ActionBarModuleItems>
+          <IconButtonBar
+            actions={[
+              {
+                label: 'Search',
+                renderIcon: Search16,
+              },
+              {
+                label: 'Filter',
+                renderIcon: Filter16,
+              },
+              {
+                label: 'View',
+                renderIcon: View16,
+              },
+            ]}
+            size="lg"
+          />
+        </ActionBarModuleItems>
+      </ActionBarModule>
+    </div>
+
+    <div className={`${settings.prefix}--col-span-100`}>
+      <CardAreaModule>
+        {({ getLayoutProps }) => (
+          <Row narrow>
+            <Column>
+              <SummaryCard {...getLayoutProps()}>
+                <SummaryCardHeader title="Threat actor" />
+              </SummaryCard>
+            </Column>
+            <Column>
+              <SummaryCard {...getLayoutProps()}>
+                <SummaryCardHeader title="Threat report" />
+              </SummaryCard>
+            </Column>
+            <Column>
+              <SummaryCard {...getLayoutProps()}>
+                <SummaryCardHeader title="IP report" />
+              </SummaryCard>
+            </Column>
+            <Column>
+              <SummaryCard {...getLayoutProps()}>
+                <SummaryCardHeader title="Vulnerability report" />
+              </SummaryCard>
+            </Column>
+          </Row>
+        )}
+      </CardAreaModule>
+    </div>
+
+    <div className={`${settings.prefix}--col-span-100`}>
+      <RowWithContainer narrow>
```

Comment:
We should now avoid using `RowWithContainer` and instead rely on the CSS grid classes.

For context, this was created as a temporary workaround for the flex grid implementation and will likely cause unintended side effects with the CSS grid implementation - my hope is that with this work we can also remove these workarounds from the codebase.